__tests__/contact/__snapshots__/contact.query.test.js.snap

```diff
@@ -1,54 +1,8 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Contacts queries #2 Should receive selected contact 1`] = `
-Object {
-  "__typename": "Contact",
-  "address": Array [
-    Object {
-      "__typename": "Language",
-      "lang": "uk",
-      "value": "Вулиця 4",
-    },
-    Object {
-      "__typename": "Language",
-      "lang": "en",
-      "value": "Street 4",
-    },
-  ],
-  "email": "test@test.com",
-  "images": Array [
-    Object {
-      "__typename": "LanguageImageSet",
-      "value": Object {
-        "__typename": "ImageSet",
-        "medium": "medium.jpg",
-      },
-    },
-    Object {
-      "__typename": "LanguageImageSet",
-      "value": Object {
-        "__typename": "ImageSet",
-        "medium": "medium.jpg",
-      },
-    },
-  ],
-  "link": "https://testURL.com",
-  "openHours": Array [
-    Object {
-      "__typename": "Language",
-      "lang": "uk",
-      "value": "ПН ...",
-    },
-    Object {
-      "__typename": "Language",
-      "lang": "en",
-      "value": "FR ...",
-    },
-  ],
-  "phoneNumber": "1241241242144",
-}
-`;
-
+<<<<<<< HEAD
+exports[`Contacts queries should receive selected contact 1`] = `null`;
+=======
```

Comment:
Resolve conflicts or remove this snapshot

Reply:
fixed